### PR TITLE
Faq   expandable questions and topics

### PR DESCRIPTION
**Pull request by mset1**

### CODE REVIEW

File: CaPPMS/Shared/FaqQuestion.razor [hunk 1]
```diff
@@ -0,0 +1,43 @@
+﻿@using CaPPMS.Model
+@namespace CaPPMS.Shared
+
+<div class="panel panel-default border @CollapseQuestion">
+    <div class="panel-body">
+        <div class="container-fluid">
+            <div class="row">
+                <div>
+                    <button @onclick="e => this.IsCollapsedReply = !this.IsCollapsedReply">
+                        @(this.IsCollapsedReply ? "+" : "-")
```
Comment (acaber) [inline CaPPMS/Shared/FaqQuestion.razor:10]: Cool! Looks good

**Review by acaber — APPROVED** [review]

File: CaPPMS/CaPPMS.csproj [hunk 2]
```diff
@@ -26,11 +26,17 @@
     <PackageReference Include="Microsoft.AspNetCore.Authentication.AzureAD.UI" Version="5.0.8" />
     <PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="5.0.9" NoWarn="NU1605" />
     <PackageReference Include="Microsoft.AspNetCore.Authentication.OpenIdConnect" Version="5.0.8" NoWarn="NU1605" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="5.0.9" />
```
Comment (willywil548) [inline CaPPMS/CaPPMS.csproj:29]: There is not a backing database by design

File: CaPPMS/Shared/FaqQuestion.razor [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+                FaqInformation="@faqInfo">
+
+            </FaqReply>
+        </div>
+    </div>
+</div>
+
+@code {
+    [Parameter]
+    public string CollapseQuestion { get; set; } = "collapse";
```
Comment (willywil548) [inline CaPPMS/Shared/FaqQuestion.razor:26]: Does this need to be public?

File: CaPPMS/Shared/FaqReply.razor [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+            <div class="row">
+                <div>Placeholder Reply for When We Implement Replying to Questions</div>
+            </div>
+        </div>
+    </div>
+</div>
+
+@code {
+    [Parameter]
+    public string CollapseReply { get; set; } = "collapse";
```
Comment (willywil548) [inline CaPPMS/Shared/FaqReply.razor:16]: Same here I guess.

**Review by willywil548 — COMMENTED** [review]
Left a few comments. It can go either and they are just comments

File: CaPPMS/CaPPMS.csproj (hunk 2 above)
Comment (mset1) [reply CaPPMS/CaPPMS.csproj:29]: I'm actually not sure what this file is exactly, I just know that when I opened vs, it changed it. I could just not include this file's changes.

File: CaPPMS/Shared/FaqQuestion.razor (hunk 3 above)
Comment (mset1) [reply CaPPMS/Shared/FaqQuestion.razor:26]: When I make this private, the editor yells at me, so I made it public so it would stop

**Pull request merged**